reference/7.2/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -133,6 +133,34 @@ provider `Alias:`. The `Get-Location` cmdlet displays the complete path for `Ali
 **C:\TestDir\AliasNames.txt**. The `Get-Content` cmdlet uses the **Path** parameter and displays the
 file's content in the PowerShell console.
 
+### Example 5: Set file output width for entire script
+
+This example uses `$PSDefaultParameterValues` to set the `Width` parameter for all invocations of
+`Out-File` and the redirection operartors (`>` and `>>`) to 2000.  This is an easy way to ensure
+that everywhere in a script you output table formatted data to file, PowerShell will use a line
+width of 2000 instead of a line width determined by the PowerShell host's console width.
+
+```powershell
+function DemoDefaultOutFileWidth() {
+    try {
+        $PSDefaultParameterValues['out-file:width'] = 2000
+
+        $logFile = "$pwd\logfile.txt"
+
+        Get-ChildItem Env:\ > $logFile
+
+        Get-Service -ErrorAction Ignore | Format-Table -AutoSize | Out-File $logFile -Append
+
+        Get-Process | Format-Table Id,SI,Name,Path,MainWindowTitle >> $logFile
+    }
+    finally {
+        $PSDefaultParameterValues.Remove('out-file:width')
+    }
+}
+
+DemoDefaultOutFileWidth
+```
+
```

Comment:
Add a link to the about topic.

```
For more information about `$PSDefaultParameterValues`, see [about_Preference_Variables](/powershell/module/microsoft.powershell.core/about/about_preference_variables#psdefaultparametervalues)
```

Reply:
I've added this link but while looking the rest of this help topic, it seems the link can be shortened to `../Microsoft.Powershell.Core/About/about_preference_variables#psdefaultparametervalues`.  if that's not right, let me know.

Reply:
I see that you fixed the link by adding the file extension. Links are tricky. The target can be a file path, which requires the `.md` extension. Or they can be a URL path which must NOT have the file extension. Also file paths can be filesystem relative. But URL paths cannot. They can be site relative, which is the format I used in the suggestion. But I like the changes you made.